### PR TITLE
More clarifications on FC channel claim

### PR DESCRIPTION
**Pull request by norkans7**
BEFORE
Facebook
Monosnap Connect FreshChat 2024-10-17 15-55-27
FreshChat
Monosnap Connect FreshChat 2024-10-17 18-17-24

AFTER
Facebook
Monosnap Connect Facebook 2024-10-17 23-06-37

FreshChat
Monosnap Connect FreshChat 2024-10-17 23-06-26

### CODE REVIEW

File: static/css/tailwind.css [hunk 1]
```diff
@@ -1879,6 +1848,11 @@ code {
   color: #2980b9;
 }
 
+ol.steps,
+ul.steps {
+  line-height: 2;
+}
+
 .card {
```
Comment (norkans7) [inline static/css/tailwind.css:1856]: these are the only additions, other changes are auto-formatting

File: static/css/tailwind.css [hunk 2]
```diff
@@ -1879,6 +1848,11 @@ code {
   color: #2980b9;
 }
 
+ol.steps,
+ul.steps {
+  line-height: 2;
+}
+
```
Comment (norkans7) [inline static/css/tailwind.css:1855]: these are the only additions, other changes are auto-formatting

File: static/css/tailwind.css [hunk 3]
```diff
@@ -563,11 +563,11 @@ video {
 }
 
 .trans-border {
-  box-shadow: 0 0px 0px 4px rgba(0, 0, 0, .1);
```
Comment (rowanseymour) [inline static/css/tailwind.css:566]: @norkans7 are these changes intentional?

Comment (norkans7) [reply static/css/tailwind.css:566]: Those are are autoformatting

File: static/css/tailwind.css (hunk 2 above)
Comment (norkans7) [reply static/css/tailwind.css:1855]: @rowanseymour

File: static/scss/tailwind.scss [hunk 4]
```diff
@@ -659,7 +643,11 @@ code {
 .code {
   @apply bg-dark-alpha-30 rounded-lg px-1 py-1 leading-tight text-base font-mono inline-block mx-1 whitespace-nowrap;
   color: #2980b9;
+}
 
+ol.steps,
+ul.steps {
+  line-height: 2;
```
Comment (norkans7) [inline static/scss/tailwind.scss:650]: @rowanseymour actually these are the only additions, other changes are auto-formatting

File: static/css/tailwind.css (hunk 2 above)
Comment (norkans7) [reply static/css/tailwind.css:1855]: Actually this file is generated too, so the only changes are in `tailwind.scss`

File: static/scss/tailwind.scss (hunk 4 above)
Comment (rowanseymour) [reply static/scss/tailwind.scss:650]: since these are steps.. I think they'd look better with numbers

```
1. first step
2. second step
```

Comment (norkans7) [reply static/scss/tailwind.scss:650]: using list decimal on the list did not look good IMO and padding does not affect that
Monosnap Connect FreshChat 2024-10-17 20-51-15
Monosnap Connect Facebook 2024-10-17 20-51-26

Comment (norkans7) [reply static/scss/tailwind.scss]: Monosnap Connect Facebook 2024-10-17 23-06-37
Monosnap Connect FreshChat 2024-10-17 23-06-26
